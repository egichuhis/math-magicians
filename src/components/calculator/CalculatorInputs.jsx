import React from 'react';

const CalculatorInputs = () => (
  <div className="buttons">
    <button type="button">
      AC
    </button>
    <button type="button">
      +/-
    </button>
    <button type="button">
      %
    </button>
    <button type="button" className="ops-color">
      รท
    </button>
    <button type="button">
      7
    </button>
    <button type="button">
      8
    </button>
    <button type="button">
      9
    </button>
    <button type="button" className="ops-color">
      x
    </button>
    <button type="button">
      4
    </button>
    <button type="button">
      5
    </button>
    <button type="button">
      6
    </button>
    <button type="button" className="ops-color">
      -
    </button>
    <button type="button">
      1
    </button>
    <button type="button">
      2
    </button>
    <button type="button">
      3
    </button>
    <button type="button" className="ops-color">
      +
    </button>
    <button type="button" className="two-columns">
      0
    </button>
    <button type="button">
      .
    </button>
    <button type="button" className="ops-color">
      =
    </button>
  </div>
);

export default CalculatorInputs;
