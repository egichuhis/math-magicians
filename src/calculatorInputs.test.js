import React from 'react';
import { render } from '@testing-library/react';
import CalculatorInputs from './components/calculator/CalculatorInputs';

test('CalculatorInputs renders correctly', () => {
  const handleClick = jest.fn();
  const { container } = render(<CalculatorInputs handleClick={handleClick} />);
  expect(container).toMatchSnapshot();
});
