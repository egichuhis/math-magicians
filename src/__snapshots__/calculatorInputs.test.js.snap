// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalculatorInputs renders correctly 1`] = `
<div>
  <div
    class="buttons"
  >
    <button
      type="button"
    >
      AC
    </button>
    <button
      type="button"
    >
      +/-
    </button>
    <button
      type="button"
    >
      %
    </button>
    <button
      class="ops-color"
      type="button"
    >
      รท
    </button>
    <button
      type="button"
    >
      7
    </button>
    <button
      type="button"
    >
      8
    </button>
    <button
      type="button"
    >
      9
    </button>
    <button
      class="ops-color"
      type="button"
    >
      x
    </button>
    <button
      type="button"
    >
      4
    </button>
    <button
      type="button"
    >
      5
    </button>
    <button
      type="button"
    >
      6
    </button>
    <button
      class="ops-color"
      type="button"
    >
      -
    </button>
    <button
      type="button"
    >
      1
    </button>
    <button
      type="button"
    >
      2
    </button>
    <button
      type="button"
    >
      3
    </button>
    <button
      class="ops-color"
      type="button"
    >
      +
    </button>
    <button
      class="two-columns"
      type="button"
    >
      0
    </button>
    <button
      type="button"
    >
      .
    </button>
    <button
      class="ops-color"
      type="button"
    >
      =
    </button>
  </div>
</div>
`;
